python-backend/app/api/mines/mine/resources/mine.py

```diff
@@ -61,48 +65,85 @@ def get(self, mine_no_or_guid=None):
+                major_mine_query = Mine.query.filter(major_mine_filter)
+                mines_permit_join_query = mines_permit_join_query.intersect(major_mine_query)
+
+            # Filter by TSF, if provided
+            if tsf_filter_term == "true" or tsf_filter_term == "false":
+                tsf_filter = Mine.mine_tailings_storage_facilities != None if tsf_filter_term == "true" else \
+                    Mine.mine_tailings_storage_facilities == None
+                tsf_query = Mine.query.filter(tsf_filter)
+                mines_permit_join_query = mines_permit_join_query.intersect(tsf_query)
+
+            # Filter by region, if provided
+            if region_code_filter_term:
+                region_filter_term_array = region_code_filter_term.split(',')
+                region_filter = Mine.mine_region.in_(region_filter_term_array)
+                region_query = Mine.query.filter(region_filter)
+                mines_permit_join_query = mines_permit_join_query.intersect(region_query)
+
+            # Filter by commodity if provided
+            if commodity_filter_terms:
+                commodity_filter_term_array = commodity_filter_terms.split(',')
+                commodity_filter = MineTypeDetail.mine_commodity_code.in_(commodity_filter_term_array)
+                mine_type_active_filter = MineType.active_ind.is_(True)
+                commodity_query = Mine.query\
+                    .join(MineType)\
+                    .join(MineTypeDetail)\
+                    .filter(commodity_filter, mine_type_active_filter)
+                mines_permit_join_query = mines_permit_join_query.intersect(commodity_query)
+
+            # Create a filter on tenure if one is provided
+            if tenure_filter_term:
+                tenure_filter_term_array = tenure_filter_term.split(',')
+                tenure_filter = MineType.mine_tenure_type_code.in_(tenure_filter_term_array)
+                mine_type_active_filter = MineType.active_ind.is_(True)
+                tenure_query = Mine.query \
+                    .join(MineType) \
+                    .filter(tenure_filter, mine_type_active_filter)
+                mines_permit_join_query = mines_permit_join_query.intersect(tenure_query)
+
+            # Create a filter on mine status if one is provided
+            if status_filter_term:
+                status_filter_term_array = status_filter_term.split(',')
+
+                status_filter = MineStatusXref.mine_operation_status_code.in_(status_filter_term_array)
+                status_reason_filter = MineStatusXref.mine_operation_status_reason_code.in_(status_filter_term_array)
+                status_subreason_filter = MineStatusXref.mine_operation_status_sub_reason_code.in_(
+                    status_filter_term_array)
+                all_status_filter = status_filter | status_reason_filter | status_subreason_filter
+                status_query = Mine.query \
+                    .join(MineStatus) \
```

Comment:
Mine to mine_status should be 1:1 after my change so this should work. Still want @JustinMacaulay to take a look